packages/scaffold-config/src/frameworks.ts

```diff
@@ -61,9 +61,16 @@ function getBundlerDependency (bundler: WizardBundler['type'], projectPath: stri
   }
 }
 
-export const WIZARD_MOUNT_MODULES = ['cypress/react', 'cypress/react18', 'cypress/vue', 'cypress/vue2', 'cypress/angular'] as const
+export const WIZARD_MOUNT_MODULES = [
+  'cypress/react',
+  'cypress/react18',
+  'cypress/vue',
+  'cypress/vue2',
+  'cypress/angular',
+  'cypress/svelte',
+] as const
 
-export type WizardMountModule = 'cypress/react' | 'cypress/react18' | 'cypress/vue' | 'cypress/vue2' | 'cypress/angular'
+export type WizardMountModule = 'cypress/react' | 'cypress/react18' | 'cypress/vue' | 'cypress/vue2' | 'cypress/angular' | 'cypress/svelte'
```

Comment:
We can infer it to reduce duplication:

```ts
export type WizardMountModule = Awaited<ReturnType<typeof WIZARD_FRAMEWORKS[number]['mountModule']>>
```

🎉

Reply:
I was having issues trying to implement this. Seems like `Awaited` is implemented in `4.5` we are on  `4.4.4`

Reply:
Hmm can we just update to TS 4.5.0? I think it's pretty easy from 4.4.4.

Alternatively I think you can implement `Awaited` with TS - there's probably an implementation you can grab for TS <= 4.5. But updating to TS 4.5 seems like a better option?

Reply:
I've created a (draft) PR here. Decent amount of changes so didn't want to change the scope of this PR

Reply:
Going to let this change be handled in that PR, thanks @ZachJW34

Reply:
Nice, `Awaited` is super nice